liveblog.php

```diff
@@ -504,6 +588,33 @@ public static function ajax_crud_entry() {
 				'latest_timestamp' => null
 			), array( 'cache' => false ) );
 		}
+
+		// self::json_return( $entry );
```

Comment:
Code that isn't used or needed should be removed completely, rather than commenting it out. If it's needed in the future it can be restored via the Git history.